unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AvatarShape/AvatarShape.cs

```diff
@@ -105,7 +105,7 @@ private void Awake()
                 visibility,
                 avatarLOD,
                 new SimpleGPUSkinning(),
-                new GPUSkinningThrottler(),
+                Environment.i.serviceLocator.Get<IGPUSkinningThrottlerService>(),
```

Comment:
You can define 
`private Service<IGPUSkinningThrottlerService> skinningThrottlerService;`
and use `skinningThrottlerService.Ref` on this line instead

Reply:
I will change it for this and the next ones ^^

Reply:
Im not so sold in the usage of a service here. This seems like a really specific system, does it make sense to make it  available for the whole platform?

Cannot we inject an instance created in the CoreComponentsPlugin to AvatarShape?

Reply:
@Kinerius My only concern is having a service widely know in the environment to only being used by a specific system. To be honest this is not egregious or even a dirty implementation.

PS. You can reply without having to edit my comment hahaha

Reply:
What do you suggest then? I wanted to avoid having a static instance or a singleton decoupled from our services. 
`CoreComponentsPlugin` is more for SDK features

Reply:
I had in mind sharing the throttler between avatar shapes and leave the player controller and the previews out of this. Therefore we could have the instance in the `CoreComponentPlugin`.  But yeah, having a service, although I find it uglier, makes sense to wrap them all.

Then what about this, remove the static accessor in AvatarShape and inject the dependency in CoreComponentPlugin. Ideally only the plugins should have access to the static environment and the sub-systems should have their needs injected.

Reply:
I think resolving `skinningThrottlerService` from `Awake` is a bigger problem than the global availability of the service itself considering that we don't have a mechanism of scopes.

Reply:
I changed it to use Avatar factory instead: now it looks much better imo